### ISSUE TITLE
Going from 8:ths to 16:ths is annoying

### ISSUE THREAD

**Issue by eriksvedang** [hisekaldma/Polyhymnia#7]
This is more of a thought than a bug.

Let's say you have 

```
Melody = C _ C _ D F E F
```

And you want to add a little extra detail at the final F and split it into two notes of the same length while keeping the rest of the notes as they are. Perhaps there could be some clever way to try that out without having to go crazy with the underscores, i.e.:

```
Melody = C _ _ _  C _ _ _  D _ F _  E _ F G
```

I'm not sure if it's a good idea but maybe something like

```
Melody = C _ C _ D F E (F G)
```

or

```
Melody = C _ C _ D F E , F G
```

To make a triple note would be as easy as this:

```
Melody = C C C (D E F)
```

Basically it would mean "group these notes together and treat them as one other note in this pattern". Kinda the opposite of the bar. I think it would be useful!

**Comment by hisekaldma** [comment]
Funny. That's exactly how Tidal does it. And it's definitely useful. I'll put it on the list of nice ideas.

**Comment by eriksvedang** [comment]
Awesome. Never heard of Tidal but that looks cool too :)

**Comment by hisekaldma** [comment]
It's one of the inspirations. Also, it's built on Haskell so it might be your kind of thing.
